app/Enums/FeatureFlag.php

```diff
@@ -41,6 +41,9 @@
 
 enum FeatureFlag: string
 {
+    case GdprBannerCustomization = 'gdpr_banner_customization';
```

Comment:
Is `GdprBannerCustomization` being used anywhere anymore? If not, let's remove it